Auroracoin endpoint [/aur]

Loknar opened the pull request:
New endpoint, exposes various information from [ISX.is](https://isx.is/about.php), an ISK <-> AUR marketplace.

### Checklist

- [x] Write tests
- [ ] Write documentation

I assume that the `docs.apis.is` site is still in this repo under the branch `gh-pages`, if so I'll probably add documentation in another pull request ..

Loknar commented:
:'(

kristjanmik commented:
Hi @Loknar, sorry for the delay.
I'm getting an error when querying the /aur endpoint, there is always an exception in the formatStatsData callback. The data being returned there is not in a json format, more like in a gzip format or some other binary format.

```
error,data null  V��R����5h
                           M,b��B��O8E~"��J�MhT���6��'�i&�E_H�/����
4U�a~6�B�;���R_��lŷ��-�
*Tp��Ah��ȝc�: 3<_7 ���eV�����A0�~����1��K�"�|?�#����"܎�y}�.�z��<��)]绝�U�F-���
21 Jun 12:19:06 - uncaughtException: 63470cb1728b49e8a0bb7301be3b4961
```

Can you check if you are getting the same results?

Loknar commented:
Can you try it now? I was mimicing browser behaviour when I added these headers, not fully sure why it wasn't failing then but is failing now, might have something to do with allowed encoding/compression options.

Removed the mimic headers.

kristjanmik commented:
Thank you @Loknar for taking the time to fix the bug <3
This LGTM, I'm merging this!
